package.json

```diff
@@ -6,5 +6,9 @@
   },
   "scripts": {
     "prepare": "husky install"
+  },
+  "dependencies": {
+    "date-fns": "^2.25.0",
+    "react-big-calendar": "^0.38.0"
```

Comment:
Is this installed in the root directory? It should be installed in the client package file.